### ISSUE TITLE
Solution to settings not opening

### ISSUE THREAD

**Issue by Byproduct** [paradoxxxzero/gnome-shell-system-monitor-applet#472]
Linux and github newbie here, sorry if this is already covered somewhere.

Just wanted to say in case you get the monitor working, but settings not opening, here's a solution.
Even though it apparently sounds uninuitive (clutter should already be included?), try:
sudo apt-get install gir1.2-clutter-1.0 gir1.2-clutter-gst-3.0 gir1.2-gtkclutter-1.0

Fixed it for me. I found the solution here: https://github.com/micheleg/dash-to-dock/issues/398.

I have a fresh Ubuntu 18.04 with nothing weird installed, so I'm guessing someone else will have the same problem too, and the solution was a bit difficult to find.

Cheers!

**Comment by chrisspen** [comment]
Interesting. `gir1.2-clutter-1.0` is already in our list of required packages in the README. Are you sure the other two are necessary?

**Comment by Byproduct** [comment]
> Interesting. `gir1.2-clutter-1.0` is already in our list of required packages in the README. Are you sure the other two are necessary?

I did follow the readme, but could've of course messed something up, so... 80% sure? 

I'll check again to verify if I have to reinstall ubuntu at some point.

**Comment by de-vries** [comment]
> Interesting. `gir1.2-clutter-1.0` is already in our list of required packages in the README. Are you sure the other two are necessary?

I just installed ubuntu 18.10 and added your extension via the gnome-shell-extension-website. I installed `gir1.2-gtop-2.0` and `gir1.2-networkmanager-1.0` because an error message said so. Then most of the extension worked but I had the same problem as OP. and that brought me here.

TL;DR: I only installed `gir1.2-clutter-1.0` and all is working fine.

Btw: there is no mention of  `gir1.2-clutter-1.0` in the readme located at `~/.local/share/gnome-shell/extensions/system-monitor@paradoxxx.zero.gmail.com/README`
`

`

**Comment by zyphlar** [comment]
Can confirm, the popup error message after install mentions gtop and networkmanager but does not mention clutter. Also, the popup cannot be copy-pasted (in Ubuntu 18.04) so it's frustrating to memorize and type into terminal. Mentioning all these dependencies directly on the Gnome Extensions website, and in the popup with Gnome restart instructions. would be greatly appreciated. (Those of us upgrading from 16.04 with indicator-multiload have already ju)
